plugin/storage/es/spanstore/reader.go

```diff
@@ -124,6 +127,8 @@ func (s *SpanReader) GetTrace(traceID model.TraceID) (*model.Trace, error) {
 func (s *SpanReader) readTrace(traceID string, traceQuery *spanstore.TraceQueryParameters) (*model.Trace, error) {
```

Comment:
not related to this diff, but needs to be fixed. Why does this function take TraceQueryParameters? It has nothing to do with what that type actually represents. Should simply pass upper and lower time boundary